### DISCUSSION TITLE
Train.py missing !!

### DISCUSSION THREAD

**Discussion by SherwinSamuel23** [PaddlePaddle/PaddleOCR#12652] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Windows 10
- 版本号/Version：PaddlePaddle：2.5.1  PaddleOCR：2.7.0.3 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：After installing Paddleocr and paddlepaddle package unable to find train.py file for kie training. Under which folder we can find train.py

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by SWHL** [comment]
If the problem still exists now, please try to clone the main branch code of the repo, and try agin.
